rest-api-spec/src/main/resources/rest-api-spec/api/logs.json

```diff
@@ -0,0 +1,57 @@
+{
+  "logs":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/master/logs.html",
+      "description":"Provides a simple JSON API to ingest log events into Elasticsearch."
+    },
+    "stability":"experimental",
+    "visibility":"public",
+    "headers":{
+      "accept": [ "application/json"],
+      "content_type": ["application/x-ndjson"]
+    },
+    "url":{
+      "paths":[
+        {
+          "path":"/_logs",
+          "methods":[
+            "POST"
+          ]
+        },
+        {
+          "path":"/_logs/{dataset}",
+          "methods":[
+            "POST"
+          ],
+          "parts":{
+            "dataset":{
+              "type":"string",
+              "description":"Default index for items which don't provide one"
```

Comment:
I'm leaving some comments on this PR which do not have to be addressed now but is rather a note for later when we get into the details of implementation.

For later: We need to discuss the exact descriptions for users here. Index sounds off. Below you have a different one.